python/equistore-operations/equistore/operations/_dispatch.py

```diff
@@ -402,7 +402,7 @@ def rand_like(array, shape=None, requires_grad=False):
         raise TypeError(UNKNOWN_ARRAY_TYPE)
 
 
-def to(array, backend: str = None, dtype=None, device=None, requires_grad=False):
+def to(array, backend: str = None, dtype=None, device=None, requires_grad=None):
```

Comment:
I think it would be nicer long term to have separate `to_torch` and `to_numpy` functions, taking whichever parameter they need to. This makes it easier to add other backends later.